packages/buendia-utils/data/usr/bin/buendia-reconfigure

```diff
@@ -66,10 +66,11 @@ if [ -n "$1" ]; then
         reconfigure_package $package
     done
 else
-    for config in /usr/share/buendia/config.d/*; do
-        if [ -f $config -a -x $config ]; then
-            reconfigure_package $(basename $config)
-        fi
+    # Find executable scripts in config.d and sort them
+    for config in $(find /usr/share/buendia/config.d -type f -executable | sort); do
```

Comment:
Why find + sort — doesn't regular globbing with * produce the results in sorted order?  Just a minor thing — I'm curious if my understanding of globbing is wrong and there's something to learn here.

Reply:
Oops, [you are right](https://www.gnu.org/software/bash/manual/html_node/Filename-Expansion.html):

> Bash scans each word for the characters ‘*’, ‘?’, and ‘[’. If one of these characters appears, then the word is regarded as a pattern, and replaced with an alphabetically sorted list of filenames matching the pattern.

So this `find | sort` pipeline is much ado about nothing, really. I think I should put it back the way it was for clarity.

Reply:
Fixed by a7c19920